src/test/java/com/avispl/symphony/dal/communicator/aver/ptz/AverPTZUtilsTest.java

```diff
@@ -0,0 +1,613 @@
+/*
+ * Copyright (c) 2021 AVI-SPL Inc. All Rights Reserved.
+ */
+package com.avispl.symphony.dal.communicator.aver.ptz;
+
+import static com.avispl.symphony.dal.communicator.aver.ptz.AverPTZUtils.buildSendPacket;
+import static com.avispl.symphony.dal.communicator.aver.ptz.AverPTZUtils.convertOneByteNumberToTwoBytesArray;
+import static org.junit.Assert.assertArrayEquals;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+
+import org.junit.Test;
+
+import com.avispl.symphony.dal.communicator.aver.ptz.enums.payload.Category;
+import com.avispl.symphony.dal.communicator.aver.ptz.enums.payload.PayloadType;
+import com.avispl.symphony.dal.communicator.aver.ptz.enums.payload.command.Command;
+import com.avispl.symphony.dal.communicator.aver.ptz.enums.payload.command.CommandType;
+import com.avispl.symphony.dal.communicator.aver.ptz.enums.payload.param.AEMode;
+import com.avispl.symphony.dal.communicator.aver.ptz.enums.payload.param.BGainControl;
+import com.avispl.symphony.dal.communicator.aver.ptz.enums.payload.param.BacklightStatus;
+import com.avispl.symphony.dal.communicator.aver.ptz.enums.payload.param.FocusControl;
+import com.avispl.symphony.dal.communicator.aver.ptz.enums.payload.param.FocusStatus;
+import com.avispl.symphony.dal.communicator.aver.ptz.enums.payload.param.PanTiltDrive;
+import com.avispl.symphony.dal.communicator.aver.ptz.enums.payload.param.PowerStatus;
+import com.avispl.symphony.dal.communicator.aver.ptz.enums.payload.param.PresetControl;
+import com.avispl.symphony.dal.communicator.aver.ptz.enums.payload.param.RGainControl;
+import com.avispl.symphony.dal.communicator.aver.ptz.enums.payload.param.SlowPanTiltStatus;
+import com.avispl.symphony.dal.communicator.aver.ptz.enums.payload.param.SlowShutterStatus;
+import com.avispl.symphony.dal.communicator.aver.ptz.enums.payload.param.WBMode;
+import com.avispl.symphony.dal.communicator.aver.ptz.enums.payload.param.ZoomControl;
+
+/**
+ * Unit test for AverPTZ Utils - method build send string
+ * Build send string which match expected string
+ *
+ * @author Harry
+ * @version 1.0
+ * @since 1.0
+ */
+public class AverPTZUtilsTest {
+	int cameraID = 1;
+	int sequenceNumber = 1;
+
+	/**
+	 * Test AverPTZUtils#buildSendPacket success
+	 * Expect build a control command for power which match expected packet
+	 */
+	@Test
+	public void testBuildSendPacketPower() {
+		// Power on
+		byte[] expectedPacketPowerOn = new byte[] { 0x01, 0x00, 0x00, 0x06, 0x00, 0x00, 0x00, 0x01, (byte) 0x81, 0x01, 0x04, 0x00, 0x02, (byte) 0xFF };
+		byte[] actualPacketPowerOn = buildSendPacket(cameraID, sequenceNumber, PayloadType.COMMAND.getCode(), CommandType.COMMAND.getCode(), Category.CAMERA.getCode(),
+				Command.POWER.getCode(), PowerStatus.ON.getCode());
+		sequenceNumber++;
+		assertArrayEquals(expectedPacketPowerOn, actualPacketPowerOn);
```

Comment:
If this is failed, the below test case won't be executed, right?
and is there any relation about the result between 2 test case to make it in one test?